geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -2373,7 +2398,7 @@ public void memberDeparted(InternalDistributedMember theId, boolean crashed, Str
           message.setCrashed(crashed);
           message.setAlertListenerExpected(true);
           message.setIgnoreAlertListenerRemovalFailure(true); // we don't know if it was a listener
-                                                              // so
+          // so
```

Comment:
Cleanup wacky comment format please.

Reply:
I usually take those trailing comments and move them above the line they are commenting, so that's what I recommend doing here.

Reply:
Yeah, I do the same. Or if the comment doesn't provide value delete it or rewrite it.